src/app/app.component.html

```diff
@@ -1,20 +0,0 @@
-<!--The content below is only a placeholder and can be replaced.-->
```

Comment:
Curious why you renamed this and related files to `app.container`? I understand you want to indicate it's a `container` component type, however, I think this is one time you can forgo that rule. This will keep things consistent with then angular CLI setup and not throw off others.

Reply:
ok will do.